build.gradle

```diff
@@ -86,7 +86,7 @@ jacocoTestCoverageVerification {
         rule {
             limit {
                 counter = 'LINE'
-                minimum = 0.80
+                minimum = 0.78
```

Comment:
Tak nie róbmy. Nie można dostosowywać zasad do kodu. To kod powinien być dostosowany do zasad, które przyjęliśmy.

Jeśli wprowadzenie InvoiceConfiguration sprawia, że nie jesteś w stanie zapewnić 80% pokrycia testami (i biorąc pod uwage fakt, że InvoiceConfiguration to nie jest coś do chcemy na siłę testować) to spróbuj wykluczyć tę klasę z obliczania pokrycia.

Tutaj znajdziesz odpowiedź na pytanie "jak" :)
https://stackoverflow.com/questions/29887805/filter-jacoco-coverage-reports-with-gradle